docs/extensibility/display-section.md

```diff
@@ -396,6 +397,32 @@ Text widgets render values as a simple text. This is the default behavior for al
 
 Block widgets are more complex layouts and you must use them only in the details body.
 
+### Alert
+
+Alert widgets display values using predefined types.
+
+#### Widget-specific parameters
+
+- **source** - information what has to be displayed.
+- **disableMargin** - an optional boolean which disables the margin outside the alert body.
+- **type** - type of alert:
+  - **information** - Defalut
+  - **warning**
+  - **error**
+  - **success**
```

Comment:
i'm not sure if listing all the fields as separate entries is necessary. Maybe just do something like this?
```suggestion
- **type** - type of alert. One of **information**, **warning**, **error** or **success**. By default it's set to **information**.
```